tests/client/getCurrentOrder.test.js

```diff
@@ -0,0 +1,17 @@
+// import {getCurrentOrder} from '../../client/apiClient'
```

Comment:
This test was written in another PR. You can remove this file since it isn't in scope for this PR anyway.